internal/prompt/prompt.go

```diff
@@ -366,6 +367,56 @@ func Seed(reader *bufio.Reader) (seed []byte, imported bool, err error) {
 	}
 }
 
+// ImportedAccounts prompts for any additional account names and xpubs to
+// import at wallet creation.
+func ImportedAccounts(reader *bufio.Reader, params *chaincfg.Params) (names []string, xpubs []*hdkeychain.ExtendedKey, err error) {
+	accounts := make(map[string]struct{})
+	accounts["default"] = struct{}{}
+
+	for {
+		fmt.Printf("Do you have additional account extended public keys to import? " +
```

Comment:
yeah it's modeled after the style of prompting done by the openbsd installer. when asked to create a user account, it just takes a username, or "done", and prints a snarky line if you answer "yes".  I left the snarky comment out here though.

I don't see why anyone would create an imported account named "y" or "yes".

Reply:
I don't see anything logically wrong code wise, but I don't really care for it tbh.  It's just asking for a user to do something dumb when a fairly simple rewording or logic change could prevent it entirely and there are even existing funcs for prompting for bools iirc (`promptListBool` I think?)

Anyway, I'll approve it since there isn't a logical bug as far as I can tell.